packages/cache/jsconfig.json

```diff
@@ -0,0 +1,24 @@
+// This file can contain .js-specific Typescript compiler config.
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+
+    "noEmit": true,
+/*
+    // The following flags are for creating .d.ts files:
+    "noEmit": false,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+*/
```

Comment:
Can we remove this? My usual inclination is to not check in commented code

Reply:
I copied from another package.  I think we should leave them the same until there is a clear reason to diverge.